cmd/syncthing/monitor.go

```diff
@@ -181,6 +186,43 @@ func copyStderr(stderr io.ReadCloser, dst io.Writer) {
 					panicFd.WriteString(line)
 				}
 				stdoutMut.Unlock()
+
+				if optPanicReporting {
+					// Prepare query for collecting error log
+					type Report struct {
+						Error       string
+						Version     string
+						LongVersion string
+						Os          string
+						Arch        string
+					}
+					report := &Report{Error: line[:len(line)-1],
+						Version:     Version,
+						LongVersion: LongVersion,
+						Os:          runtime.GOOS,
+						Arch:        runtime.GOARCH}
+					b, err := json.Marshal(report)
+					if err == nil {
+						var jsonString = `docs=` + string(b)
+						// Send error log
+						resp, err := http.Post("http://188.166.50.65:27080/logsdb/logs/_insert", "application/x-www-form-urlencoded", strings.NewReader(jsonString))
```

Comment:
So... What's the server side here? It seems to me that this should better be somewhat more generic, i.e. `http.Post("https://data.syncthing.net/panic")` and the receiving side can sort out what to do on the backend (i.e. push it into some database etc). Also, HTTPS and a DNS name. And hence push it just as a JSON blob of type `application/json` or something since all our other POST requests look like that?